src/components/Table.js

```diff
@@ -0,0 +1,125 @@
+                        <td>{ italian }</td>
+                        <td>{ arabic }</td>
+                        <td>{ serbian }</td>
+                        <td>{ croatian }</td>
+                        <td>{ russian }</td>
+                        <td>{ german }</td>
+                        <td>{ hebrew }</td>
+                        <td>{ french }</td>
+                        <td>{ hungarian }</td>
+                        <td>{ slovak }</td>
+                        <td>{ spanish }</td>
+                        <td>{ portuguese }</td>
+                        <td>{ turkce }</td>
+                        <td>{ greek }</td>
+                        <td>{ romanian }</td>
+                    </tr>
+                )
+            })
+        } else {
+            return (
+                <tr>
+                    <td>{ "loading" }</td>
+                </tr>
+            )
+        }
+    }
+
+    componentDidMount() {
+        fetch( 'https://cmlgbackend.wdcc.co.nz/translations' )
+            .then( results => {
+                return results.json();
+            })
+            .then( data => {
+                let sortedListOfWords = [];
+                let translationsForOneWord = [];
+                let dataIndex;
+                let currentData;
+
+                for ( dataIndex = 0; dataIndex < data.length; dataIndex++ ) {
+                    currentData = data[ dataIndex ];
+
+                    // Check if the translated word is in the correct column (under the correct language). So if the
+                    // English translation for the word is not under English, throw an exception.
+                    // + 1 infront of translationsForOneWord.length because the index starts from 0, whereas language_id
+                    // starts from 1.
+                    if ( translationsForOneWord.length + 1 === currentData.language_id ) {
+                        translationsForOneWord[ translationsForOneWord.length ] = currentData.name;
+                    } else {
+                        throw new Error( "The translated word does not match the language." );
+                    }
+
+                    const numberOfLanguages = 18;
+
+                    // When the word is translated to all languages, add translationsForOneWord into sortedListOfWords.
+                    // Empty translationsForOneWord so a new translationsForOneWord can be made for a new word.
+                    if ( translationsForOneWord.length === numberOfLanguages ) {
+                        sortedListOfWords[ sortedListOfWords.length ] = translationsForOneWord;
+                        translationsForOneWord = [];
+                    }
+                }
+
+                this.setState({
+                    translationData: sortedListOfWords
+                });
+            })
+    }
+
+    render() {
+        return (
+            <table className="table table-striped">
+                <thead>
+                <tr>
```

Comment:
can we have an indentation here for the <tr> element? since <tr> is belong to <thead>